### PR TITLE
added ProfileModelTest

### PR DESCRIPTION
**Pull request by Pat-7-626**
I added ProfileModelTest for testing Profile model in models.py

### CODE REVIEW

**Review by knilios — APPROVED** [review]
Seems ok to me.

File: webpage/tests/test_recipestep.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def setUpTestData(cls):
             recipe=cls.recipe)
 
     def test_recipe_step_create(self):
-        """Test if the RecipeStep object is created correctly."""
+        """Test if the recipeStep object is created correctly."""
```
Comment (SariyaP) [inline webpage/tests/test_recipestep.py:34]: Is this typo or is it already correct?

Comment (Pat-7-626) [reply webpage/tests/test_recipestep.py:34]: correct

**Review by SariyaP — APPROVED** [review]
You can merge

**Pull request merged**